docs/python-client.rst

```diff
@@ -11,6 +11,16 @@ much easier to upload a large, nested hierarchy of data from a local directory
 to Girder, and also makes it much easier to download a large, nested hierarchy
 of data from Girder to a local directory.
 
+Versioning
+----------
+
+The Girder python client is semanticly versioned separately from the Girder
+server, but not fully independently because the client and server are coupled.
+The design intent is that every version of the client should work with every
+version of the server, modulo bugfixes, though this is not an explicit
```

Comment:
"modulo bugfixes" may not be a great phrase to use for the audience. I think we can remove the phrase here, since the line "though this is not an explicit guarantee" kind of captures that sentiment anyway.